frontend/src/components/TransferScreenComponents/DropDownAccountsList.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Text, Icon, Picker, Card, CardItem,
+} from 'native-base';
+
+export default function DropDownAccountsList({ accounts, transferAccountSelection }) {
+  const [selected, setSelect] = React.useState(undefined);
```

Comment:
```suggestion
  const [selected, setSelected] = React.useState(undefined);
```
State name and setter should be same, also selected what? naming is not clear enough